inference-engine/samples/classification_sample_async/main.cpp

```diff
@@ -186,22 +186,27 @@ int main(int argc, char *argv[]) {
         size_t curIteration = 0;
         std::condition_variable condVar;
 
-        inferRequest.SetCompletionCallback(
```

Comment:
how the changes are relevant to the Cancelability?

Reply:
I did use this code to  test functionality. I will revert this back.

Reply:
Fixed